static/app/utils/theme/theme.tsx

```diff
@@ -1092,13 +1092,49 @@ const iconSizes: Record<Size, string> = {
 
 const space = {
   none: '0px',
+  /**
+   * `2px`
```

Comment:
fwiw you can see the size when you use your LSP's hover capability

<img width="399" height="212" alt="image" src="https://github.com/user-attachments/assets/7e5bed6e-a1e0-47a0-bd11-a3559f7bdd99" />

Reply:
Good point! Removed in [c314614](https://github.com/getsentry/sentry/pull/96003/commits/c314614925580b685accfbad681da5974da3ac7a)